Sun May  4 11:28:45 FET 2014  Sergei Trofimovich <slyfox@community.haskell.org>
  * fix 'die' clash with forthcoming base from ghc-7.9
  
  Current ghc-HEAD clashes thusly:
  > [5 of 6] Compiling System.Console.CmdLib.Record ( System/Console/CmdLib/Record.hs, dist/dist-sandbox-90719533/build/System/Console/CmdLib/Record.o )
  >
  > System/Console/CmdLib/Record.hs:237:33:
  >     Ambiguous occurrence die
  >     It could refer to either System.Console.CmdLib.Command.die,
  >                              imported from System.Console.CmdLib.Command at System/Console/CmdLib/Record.hs:5:1-36
  >                              (and originally defined
  >                                 at System/Console/CmdLib/Command.hs:291:1-3)
  >                           or System.Exit.die,
  >                              imported from System.Exit at System/Console/CmdLib/Record.hs:15:1-18
diff -rN -u old-cmdlib/System/Console/CmdLib/Command.hs new-cmdlib/System/Console/CmdLib/Command.hs
--- old-cmdlib/System/Console/CmdLib/Command.hs	2014-05-04 11:30:46.645611217 +0300
+++ new-cmdlib/System/Console/CmdLib/Command.hs	2014-05-04 11:30:46.648611217 +0300
@@ -14,7 +14,7 @@
 import Control.Monad( when, forM_ )
 import Control.Exception ( evaluate, catch, SomeException )
 import System.IO( hPutStrLn, stderr )
-import System.Exit
+import System.Exit ( exitWith, ExitCode(..) )
 import Prelude hiding ( catch )
 
 -- | How to process options for a command. See "optionStyle" for details.
diff -rN -u old-cmdlib/System/Console/CmdLib/Record.hs new-cmdlib/System/Console/CmdLib/Record.hs
--- old-cmdlib/System/Console/CmdLib/Record.hs	2014-05-04 11:30:46.645611217 +0300
+++ new-cmdlib/System/Console/CmdLib/Record.hs	2014-05-04 11:30:46.648611217 +0300
@@ -12,7 +12,7 @@
 import Control.Monad ( when )
 import Control.Monad.State( evalState, get, put, State, execState )
 import Data.Maybe( fromMaybe, fromJust )
-import System.Exit
+import System.Exit ( exitWith, ExitCode(..) )
 
 import System.Console.CmdLib.Flag
 
